EXTRACTORS/FILE/READ_CSV/READ_CSV.py

```diff
@@ -9,10 +9,19 @@ def READ_CSV(
     """
     The READ_CSV node reads a .csv file from disk or a URL, and then returns a dataframe.
 
+    Inputs
+    ------
+    default: None
+
```

Comment:
Don't include sections that aren't relevant, this node doesn't have any inputs so just exclude this